README.rst

```diff
@@ -8,3 +8,15 @@ Features
 --------
 
 * TODO
+
+
+Settings
+--------
+
+- LOGIN_REDIRECT_URL
+- LOGIN_URL
+- COLA_COOKIE_NAME
+- COLA_JWT_EXTRACTION_REGEX_PATTERN
+- COLA_COGNITO_CLIENT_ID
+- AWS_REGION_NAME
+- COLA_COGNITO_USER_POOL_ID
```

Comment:
Is it worth adding a note saying where these settings would come from?

Reply:
@rachaelcodes can you check the readme and see if you're happy with the added notes?